### PR TITLE
Address weird crash case

### PR DESCRIPTION
**Pull request by ZeldaZach**

### CODE REVIEW

File: cockatrice/src/game/player/player.h [hunk 1]
```diff
@@ -222,7 +222,7 @@ private slots:
     void actFlowT();
     void actSetAnnotation();
     void actPlay();
-    void actHide();
+    void actHide() const;
```
Comment (ebbit1q) [inline cockatrice/src/game/player/player.h:225]: what's the benefit of making this const anyway? it modifies the selected card so I'd hardly consider its behavior const even though it doesn't modify the player object directly

Comment (ZeldaZach) [reply cockatrice/src/game/player/player.h:225]: Was a recommendation from Clion; more so these methods don't modify the player, so technically they're consts.

Comment (ebbit1q) [reply cockatrice/src/game/player/player.h:225]: ok, but this function calls CardZone::removeCard which then calls Player::deleteCard, a non const function. Marking this function const while it actually indirectly does non const things is misguiding.

**Review by ebbit1q — APPROVED** [review]

**Comment by ebbit1q** [comment]
we can't have too many null checks now can we?

**Pull request merged**